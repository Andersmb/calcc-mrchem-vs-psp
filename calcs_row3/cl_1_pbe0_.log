Starting job 5688111 on c1-[21,26],c2-7,c3-[6,13],c5-7 at Thu May 19 22:08:24 CEST 2022

Copying cl_1_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying cl_1_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688111      cl_1_pbe0_        180                                             00:30:21      0:0 
5688111.bat+      batch         24        1   00:00:03          0   00:00:03   00:30:21      0:0 
5688111.ext+     extern        180        6   00:00:00          5   00:00:00   00:30:21      0:0 
5688111.0      mrchem.x        180       30   00:29:48         29   01:23:21   00:29:57      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688111                                                                          
5688111.bat+     19336K          0     19336K        0              0          0 
5688111.ext+          0          5          0        0              5          0 
5688111.0     12022644K         29 4338509141        0             19          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688111                                                                                               
5688111.bat+       16.88M               0         16.88M        0.03M                0          0.03M 
5688111.ext+        0.00M               5          0.00M            0                5              0 
5688111.0           6.75M               0          6.73M      133.69M                0         59.53M 

Job 5688111 completed at Thu May 19 22:38:45 CEST 2022
