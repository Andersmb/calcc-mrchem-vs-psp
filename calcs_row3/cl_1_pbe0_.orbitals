/cluster/projects/nn4654k/ambr/MWOrbitals/5688111
