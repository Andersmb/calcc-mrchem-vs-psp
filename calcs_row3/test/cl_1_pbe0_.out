                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Cl            :        0.000000        0.000000       -1.878293
    1      Cl            :        0.000000        0.000000        1.878293
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              17
 Beta electrons          :                                              17
 Total electrons         :                                              34
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999986002246e-01
    1    1    a          :                              1.000001606845e+00
    2    1    a          :                              1.000000013053e+00
    3    1    a          :                              9.999939751824e-01
    4    1    a          :                              9.999990992850e-01
    5    1    a          :                              1.000000802743e+00
    6    1    a          :                              1.000000217413e+00
    7    1    a          :                              1.000000125683e+00
    8    1    a          :                              9.999999925499e-01
    9    1    a          :                              9.999999926615e-01
   10    1    a          :                              9.999999999644e-01
   11    1    a          :                              1.000000000007e+00
   12    1    a          :                              9.999999997855e-01
   13    1    a          :                              9.999999866729e-01
   14    1    a          :                              9.999999923713e-01
   15    1    a          :                              1.000000004377e+00
   16    1    a          :                              1.000000019112e+00
   17    1    b          :                              9.999997460510e-01
   18    1    b          :                              1.000002679774e+00
   19    1    b          :                              1.000000000011e+00
   20    1    b          :                              1.000000000004e+00
   21    1    b          :                              1.000000013719e+00
   22    1    b          :                              9.999992963411e-01
   23    1    b          :                              1.000004967210e+00
   24    1    b          :                              1.000002828062e+00
   25    1    b          :                              1.000000296998e+00
   26    1    b          :                              9.999997052752e-01
   27    1    b          :                              9.999999999564e-01
   28    1    b          :                              9.999998987727e-01
   29    1    b          :                              1.000000023412e+00
   30    1    b          :                              9.999999864906e-01
   31    1    b          :                              9.999999923717e-01
   32    1    b          :                              9.999999977764e-01
   33    1    b          :                              1.000000019114e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                912.913652709478
 E-N energy              :            (au)              -2330.404432800838
 Coulomb energy          :            (au)                481.018841663704
 Exchange energy         :            (au)                -13.742970380694
 X-C energy              :            (au)                -42.306943744341
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 76.931548403218
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -992.521852552691
 Nuclear energy          :            (au)                 76.931548403218
---------------------------------------------------------------------------
 Total energy            :            (au)             -9.155903041495e+02
                         :      (kcal/mol)             -5.745415855784e+05
                         :        (kJ/mol)             -2.403881995620e+06
                         :            (eV)             -2.491448031426e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)               -101.093073215780
    1    1    a          :            (au)               -101.089602133655
    2    1    a          :            (au)                 -9.545732485712
    3    1    a          :            (au)                 -9.544000506663
    4    1    a          :            (au)                 -7.296129999722
    5    1    a          :            (au)                 -7.292936083037
    6    1    a          :            (au)                 -7.278045874259
    7    1    a          :            (au)                 -7.277212115531
    8    1    a          :            (au)                 -7.276096552769
    9    1    a          :            (au)                 -7.275871264126
   10    1    a          :            (au)                 -1.006598818261
   11    1    a          :            (au)                 -0.798855836737
   12    1    a          :            (au)                 -0.494563119765
   13    1    a          :            (au)                 -0.444776174783
   14    1    a          :            (au)                 -0.444469157309
   15    1    a          :            (au)                 -0.335505066442
   16    1    a          :            (au)                 -0.335253025473
   17    1    b          :            (au)               -101.093209551509
   18    1    b          :            (au)               -101.089423132840
   19    1    b          :            (au)                 -9.545814587476
   20    1    b          :            (au)                 -9.543924271028
   21    1    b          :            (au)                 -7.296357275620
   22    1    b          :            (au)                 -7.292804168868
   23    1    b          :            (au)                 -7.278049751152
   24    1    b          :            (au)                 -7.277206962848
   25    1    b          :            (au)                 -7.276094165818
   26    1    b          :            (au)                 -7.275878746755
   27    1    b          :            (au)                 -1.006601661795
   28    1    b          :            (au)                 -0.798854538443
   29    1    b          :            (au)                 -0.494564203670
   30    1    b          :            (au)                 -0.444776479086
   31    1    b          :            (au)                 -0.444468783130
   32    1    b          :            (au)                 -0.335491363586
   33    1    b          :            (au)                 -0.335251216973
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -537.657492290621
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          5.830952e+00        -915.590304149473         -9.155903e+02
    1          2.056934e-01        -919.961086643318         -4.370782e+00
    2          6.311313e-02        -920.091405301607         -1.303187e-01
    3          3.229836e-02        -920.110810417532         -1.940512e-02
    4          3.860651e-02        -920.117407904514         -6.597487e-03
    5          2.327427e-02        -920.118895876117         -1.487972e-03
    6          1.507015e-02        -920.119656609498         -7.607334e-04
    7          6.474228e-03        -920.119829112231         -1.725027e-04
    8          5.152430e-03        -920.119872353411         -4.324118e-05
    9          2.913164e-03        -920.119883034462         -1.068105e-05
   10          3.366080e-03        -920.119900487744         -1.745328e-05
   11          1.765453e-03        -920.119906445510         -5.957766e-06
   12          6.790554e-04        -920.119908190040         -1.744531e-06
   13          2.726386e-04        -920.119908602410         -4.123694e-07
   14          1.391977e-04        -920.119908744734         -1.423246e-07
---------------------------------------------------------------------------
                      SCF converged in 14 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Cl            :        0.000000        0.000000       -1.878293
    1      Cl            :        0.000000        0.000000        1.878293
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                918.952182818438
 E-N energy              :            (au)              -2342.454113845230
 Coulomb energy          :            (au)                482.618257351686
 Exchange energy         :            (au)                -13.760627060788
 X-C energy              :            (au)                -42.407156412058
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 76.931548403218
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -997.051457147952
 Nuclear energy          :            (au)                 76.931548403218
---------------------------------------------------------------------------
 Total energy            :            (au)             -9.201199087447e+02
                         :      (kcal/mol)             -5.773839553527e+05
                         :        (kJ/mol)             -2.415774470764e+06
                         :            (eV)             -2.503773712902e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)               -101.762623005901
    1    1    a          :            (au)               -101.762621811480
    2    1    a          :            (au)                 -9.585165189221
    3    1    a          :            (au)                 -9.585099909291
    4    1    a          :            (au)                 -7.322550572891
    5    1    a          :            (au)                 -7.322378647473
    6    1    a          :            (au)                 -7.309734093350
    7    1    a          :            (au)                 -7.309734088424
    8    1    a          :            (au)                 -7.309725800056
    9    1    a          :            (au)                 -7.309725787587
   10    1    a          :            (au)                 -0.961644019533
   11    1    a          :            (au)                 -0.789982241536
   12    1    a          :            (au)                 -0.489385860818
   13    1    a          :            (au)                 -0.427478987787
   14    1    a          :            (au)                 -0.427478957131
   15    1    a          :            (au)                 -0.320504643637
   16    1    a          :            (au)                 -0.320504613002
   17    1    b          :            (au)               -101.762623000845
   18    1    b          :            (au)               -101.762621818861
   19    1    b          :            (au)                 -9.585165191397
   20    1    b          :            (au)                 -9.585099907898
   21    1    b          :            (au)                 -7.322550575216
   22    1    b          :            (au)                 -7.322378649634
   23    1    b          :            (au)                 -7.309734100069
   24    1    b          :            (au)                 -7.309734089487
   25    1    b          :            (au)                 -7.309725804332
   26    1    b          :            (au)                 -7.309725792505
   27    1    b          :            (au)                 -0.961644011882
   28    1    b          :            (au)                 -0.789982232670
   29    1    b          :            (au)                 -0.489385839541
   30    1    b          :            (au)                 -0.427478984655
   31    1    b          :            (au)                 -0.427478954858
   32    1    b          :            (au)                 -0.320504641940
   33    1    b          :            (au)                 -0.320504610157
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -540.632676435066
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000        0.000002       -0.000001
 Magnitude               :            (au)                        0.000003
                         :         (Debye)                        0.000006
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :       -0.000000        0.000002       -0.000001
 Magnitude               :            (au)                        0.000003
                         :         (Debye)                        0.000006
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 29m 48s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
